raiden/api/v1/encoding.py

```diff
@@ -242,7 +242,7 @@ class Meta:
 
 
 class ChannelStateSchema(BaseSchema):
-    channel_identifier = KeccakField(attribute='identifier')
```

Comment:
`KeccakField` is unused afterwards and can be removed

Reply:
removed it